tests/Unit/MysqlToMysqlTest.php

```diff
@@ -67,7 +67,6 @@ public function testTablesOption()
             '--schema-to'   => $this->target_connection,
             '--tables'      => [$this->table_foo, $this->table_bar],
         ])
-            ->expectsConfirmation('Please confirm table list should be retrieved from target connection? (incase if source connection does not support it)', 'no')
```

Comment:
Why are you deleting this question?

In code, if tables is not empty, it must be set: https://github.com/Jagdish-J-P/migrate-db/blob/main/src/Console/Migrate.php#L307

Reply:
Ah, ok.